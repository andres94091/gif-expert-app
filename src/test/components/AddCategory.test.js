import '@testing-library/jest-dom';
import React from 'react';
import { shallow } from 'enzyme';
import { AddCategory } from '../../components/AddCategory';


describe('pruebas en <AddCategory/>', () => {

    const setCategories = () => {}
    const wrapper = shallow(<AddCategory setCategories = {setCategories}/>);
    
    test('should ', () => {
        
        expect(wrapper).toMatchSnapshot()
    })

    test('debe de cambiar la caja de texto', () => {
        
        const input  = wrapper.find('input')
        const value = 'hola mundo'
        input.simulate('change', {target: {value}})

        expect(wrapper.find('p').text().trim()).toBe(value)
    })
    
    
})
