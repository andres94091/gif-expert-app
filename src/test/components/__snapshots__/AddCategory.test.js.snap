// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en <AddCategory/> should  1`] = `
<form
  onSubmit={[Function]}
>
  <p />
  <input
    onChange={[Function]}
    type="text"
    value=""
  />
</form>
`;
